content/position/2022-08-22-postdoc-hake-ml/index.md

```diff
@@ -0,0 +1,67 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+title: "Postdoc position for machine learning in fisheries acoustics for ecological applications"
+date: 2022-08-22T00:00:00-08:00
+authors: 
+  - leewj
+  - vms16
+slug: 2022-08-22-postdoc
+draft: false
+tags: 
+summary: " "
+
+# Customize
+share: false
+markup: md
+math: true
+featured: false
+
+# Featured image
+# To use, place an image named `featured.jpg/png` in your page's folder.
+# Placement options: 1 = Full column width, 2 = Out-set, 3 = Screen-width
+# Focal point options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
+# Set `preview_only` to `true` to just use the image for thumbnails.
+image:
+  caption:
+  Placement options: 1
+  focal_point: "Center"
+  preview_only: true
+
+---
+
+### Location
+Seattle, WA, USA 
+
+### Open date
+August 22, 2023
+
+### Deadline
+Open until filled. Review of materials will start immediately.
+
+### Description
+We are looking for a highly motivated early-career scientist to join the [Echospace](https://uw-echospace.github.io/) group at the University of Washington to conduct research at the intersection of ocean acoustics and machine learning. The postdoc will develop high-throughput methodologies to extract information from large volumes of ocean acoustic data. The initial project will focus on automatic identification of fish and zooplankton in water column active acoustic data collected by ships, with expansions to integrate additional active/passive acoustic and environmental data streams from collocated moorings in the highly productive [northern California Current System](https://tos.org/oceanography/article/why-is-the-northern-end-of-the-california-current-system-so-productive). The project is in close collaboration with the NOAA Northwest Fisheries Science Center. The successful candidate will have a wide range of opportunities to develop new projects, build integrated skills across physics-based models, data-driven methods, and cloud computing, and interact with the vibrant ocean sciences and data science communities at UW and in Seattle. 
```

Comment:
I do think we should add echousounder somewhere, since this is what kind of data people will work with, at least during the first year. In fact, folks who are aware of this most probably would be most interested in the project. So I suggest either "volumes of ocean echosounder data" or later in fish and zooplankton echosounder data collected by ships. There are already statements after that that they can have different expansions.

Reply:
changed